molgenis-data-semanticsearch/src/test/java/org/molgenis/data/semanticsearch/service/impl/SemanticSearchServiceHelperTest.java

```diff
@@ -216,9 +211,10 @@ public void testFindTagsSync()
 	public void testSearchCircumflex() throws InterruptedException, ExecutionException
 	{
 		String description = "body^0.5 length^0.5";
-		Set<String> expected = Sets.newHashSet("length^0.5", "body^0.5");
+		Set<String> expected = Sets.newHashSet("length", "body", "0.5");
```

Comment:
Yes it is. 
We do use the caret character internally for boosting specific words in the query after cleaning up the strings by removing stop words and illegal characters. However we don`t allow strings to contain the caret character, that means it`d be impossible to boost a  query outside the semantic search class.